packages/gateway/src/features/fhir/paymentsAPI.ts

```diff
@@ -10,26 +10,26 @@
  */
 
 // eslint-disable-next-line import/no-relative-parent-imports
+import { AugmentedRequest } from '@apollo/datasource-rest'
 import { FHIR_URL } from '@gateway/constants'
+import { OpenCRVSRestDataSource } from '@gateway/graphql/data-source'
 import { PaymentReconciliation } from '@opencrvs/commons/types'
-import { RequestOptions, RESTDataSource } from 'apollo-datasource-rest'
 
-export default class PaymentsAPI extends RESTDataSource {
-  constructor() {
-    super()
-    this.baseURL = `${FHIR_URL}/PaymentReconciliation`
-  }
+import { parse } from 'url'
+
+const fhirPath = parse(FHIR_URL).path || ''
```

Comment:
Is the fallback intentional, meaning no path, or should error be thrown if config is missing?